### PR TITLE
Create a new case study for pywhy, causaltune, Targeting variants for maximum impact

### PR DESCRIPTION
**Pull request by AlxdrPolyakov**
Hello, I am writing to you as you are one of the contributors to py-why.github.io. I am one of the developers of CausalTune, which is currently inside PyWhy. We have previously added intros to the pywhy site as well. Now we would like to add a case study to the pywhy site. This case study is about ERUPT metric and different techniques for marketing causal inference. Added case study md file and png file, thank you!

### CODE REVIEW

**Comment by fverac** [comment]
Screenshots of changes:

image

image

Seems like it could be helpful to add a more verbose description via the "summary" attribute, for the purposes of the dedicated case studies page. Example in 10-Intro-to-Causal-AI.md

**Comment by AlxdrPolyakov** [comment]
> Screenshots of changes:
> 
> image
> 
> image
> 
> Seems like it could be helpful to add a more verbose description via the "summary" attribute, for the purposes of the dedicated case studies page. Example in 10-Intro-to-Causal-AI.md

Thank you, added description

**Comment by AlxdrPolyakov** [comment]
Added better description

**Comment by AlxdrPolyakov** [comment]
just updated link from Medium to Towards Data Science, ready to be merged

File: _case_studies/11_targeting_variants_for_maximum_impact.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Targeting variants for maximum impact using the CausalTune library
```
Comment (emrekiciman) [inline _case_studies/11_targeting_variants_for_maximum_impact.md]: "Targeting variants" sounds very technical.  Is there a way to re-word the title to make it clearer who would want to read this case study and why? In the tutorial itself, several of the phrases used in sections seem to be clearer --> "improving key business metrics", "optimizing clickthrough rates", etc.  Would one of those work for the tutorial title?

File: _case_studies/11_targeting_variants_for_maximum_impact.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+layout: page
+description: >-
+  This tutorial provides an introduction to causal AI using the CausalTune library in Python. It shows a practical example and the use of the ERUPT metric.
```
Comment (emrekiciman) [inline _case_studies/11_targeting_variants_for_maximum_impact.md]: I'd recommend emphasizing what the reader will learn and be able to do after reading this tutorial.  If the ERUPT metric is a key part of that, maybe briefly mention it like "This tutorial provides an introduction to [doing ABC] using the ERUPT metric and the CausalTune library in Python.

Alternatively, leave the ERUPT metric out of the short description and reference it in the summary below where there's a little more room to define it.

**Review by emrekiciman — CHANGES_REQUESTED** [review]

File: _case_studies/11_targeting_variants_for_maximum_impact.md (hunk 1 above)
Comment (AlxdrPolyakov) [reply _case_studies/11_targeting_variants_for_maximum_impact.md]: Thank you, changed to "Improving business metrics for maximum impact using the CausalTune library"

File: _case_studies/11_targeting_variants_for_maximum_impact.md (hunk 2 above)
Comment (AlxdrPolyakov) [reply _case_studies/11_targeting_variants_for_maximum_impact.md]: Changed to "This tutorial provides an introduction to improving business metrics using the ERUPT metric and the CausalTune library in Python. It shows a practical example and the use of the ERUPT metric for optimizing clickthrough rates."

**Comment by emrekiciman** [comment]
Thanks @AlxdrPolyakov .  As I read through this again, is "maximum impact" overpromising?  @amit-sharma any other suggestions?

**Comment by AlxdrPolyakov** [comment]
@emrekiciman changed to "for better impact", is that good enough?

**Review by emrekiciman — APPROVED** [review]

**Pull request merged**